tests/Agent/UnitTests/Core.UnitTest/Configuration/DefaultConfigurationTests.cs

```diff
@@ -239,6 +239,20 @@ public void TransactionEventsMaxSamplesOf0ShouldDisableTransactionEvents()
             Assert.IsFalse(_defaultConfig.TransactionEventsEnabled);
         }
 
+        [Test]
+        public void DisableServerConfigIsFalseByDefault()
+        {
+            Assert.IsFalse(_defaultConfig.DisableServerConfiguration);
+        }
+
+        [TestCase("true", ExpectedResult = true)]
+        [TestCase("false", ExpectedResult = false)]
+        public bool DisableServerConfigSetFromEnvironment(string environment)
+        {
+            Mock.Arrange(() => _environment.GetEnvironmentVariable("NEW_RELIC_DISABLE_SERVER_CONFIG")).Returns(environment);
```

Comment:
Could you change the env variable name to `NEW_RELIC_IGNORE_SERVER_SIDE_CONFIG` to better reflect what is going on ?